src/donationForm.js

```diff
@@ -86,14 +87,20 @@ const DonationForm = (props) => {
 
         setIsLoading(true);
 
+        // 💵 How much should be charged? Converts to cents for non-zero decimal currencies.
+        const chargeAmount = zeroDecimalCodes.includes(props.attributes.currencyCode) ? donationAmount : donationAmount * 100;
```

Comment:
Some currencies have 3 minor units. See my review comment for further details on how to get about this.

Reply:
@DevinWalker I'm confused. Did you decide this is fine or something?

Reply:
Yeah... It's not likely anyone is using a three decimal currency right now so I figured leave that for another day and get a release out.

Reply:
Normally, I would agree, but keep in mind that if someone with a currency other than 0/2 minor units uses the block it's going to charge the wrong amount to people's accounts... that's real bad. 😬

Reply:
I'll make sure I fix it asap. Just for you, and all the three decimal currencies out there. 🙂